app/views/templates/MainTemplate.scala.html

```diff
@@ -14,9 +14,12 @@
  * limitations under the License.
  *@
 
-@import config.{RenderConfig, FrontendAppConfig}
+@import config.{FrontendAppConfig, RenderConfig}
+@import models.LocalReferenceNumber
+@import resources._
 @import uk.gov.hmrc.hmrcfrontend.views.config.StandardBetaBanner
 @import uk.gov.hmrc.hmrcfrontend.views.viewmodels.timeoutdialog.TimeoutDialog
+@import views.utils.ViewUtils._
```

Comment:
Same as above. Think this might be one of the default imports defined in build.sbt